tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc

```diff
@@ -88,6 +92,32 @@ int initWeightPrePackingCacheCapacity() {
   return std::atoi(env);
 }
 
+#if defined(TAO_AARCH64)
+bool applyACLThreadPoolConfigIfNotSet() {
+  static bool status = []() {
+    // make sure oneDNN ACL thread pool is configured first.
+    // It'll be initialized inside the constructor of the engine.
+    (void)ideep::engine::cpu_engine();
+    // override ACL thread pool config if necessary.
+    if (getNumAvailableCores() == 1) {
+      // using single thread scheduler
+      arm_compute::Scheduler::set(arm_compute::Scheduler::Type::ST);
+      TAO_VLOG(1)
+          << "Enforce to use single thread schedule due to `OMP_NUM_THREADS=1`";
+    } else if (const char* scheduler_type = getenv("DISC_ACL_SCHEDULER_TYPE")) {
+      if (std::strcmp(scheduler_type, "OMP") == 0) {
```

Comment:
Nit: should we make `OMP` as default value for `DISC_ACL_SCHEDULER_TYPE`?

Reply:
Currently, the default value is CPP same as ACL. I'll do some performance study first and then change the default to `OMP` if no performance loss.